packages/vike-react-chakra/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "vike-react-chakra",
+    "version": "0.0.1",
+    "type": "module",
+    "exports": {
+        "./config": "./dist/integration/+config.js",
+        "./__internal/integration/Wrapper": "./dist/integration/Wrapper.js"
```

Comment:
Given that `vike-react-chakra` has few files, let's remove the `integration/` directory. Let's also remove it in the export path.

Reply:
Also let's `mv Wrapper.tsx +Wrapper.tsx` so that it matches what we will eject (once we finished the eject implementation).

(FYI I'll update the `vike-{react,vue}` packages to also prefix all entries with `+`.)

Reply:
Then, we can remove this line too ?
```js
// +config.ts
// ...
  Wrapper: 'import:vike-react-chakra/__internal/integration/Wrapper:Wrapper',
// ...
```

Reply:
We still need it, Vike never crawls inside `node_modules/`.

Reply:
WDYT https://github.com/vikejs/vike-react/pull/151/commits/de3e81166390282e9fd5187d805badfd0a5d9459 ?

Reply:
Perfect :sparkles: